docker-compose.yaml

```diff
@@ -6,17 +6,19 @@ services:
     volumes:
     - db_data:/var/lib/postgresql-11.2/data
   graphql-engine:
-    image: hasura/graphql-engine:v1.0.0-beta.8
+    build: 
+      context: ./database-service
+      dockerfile: ./Dockerfile
+      network: host 
+      target: app
     ports:
-    - "8080:8080"
+    - 8080:8080
     depends_on:
-    - "postgres"
+    - postgres
     restart: always
     environment:
       HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:@postgres:5432/postgres
-      HASURA_GRAPHQL_ENABLE_CONSOLE: "true" # set to "false" to disable console
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
       HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
-      ## uncomment next line to set an admin secret
-      # HASURA_GRAPHQL_ADMIN_SECRET: myadminsecretkey
 volumes:
   db_data:
```

Comment:
Here I optimize absolutely nothing as that docker-compose will never be used in production. The relevant configuration will be done in a jps manifest.

Reply:
That's because you haven't tried to run the `docker-compose` on the build server. Look at the `publishBackendDockerImage` method of the [Jenkins helper method](https://github.com/shopozor/jenkins-shared-library/blob/master/src/ch/softozor/pipeline/Helpers.groovy). On my dev laptop, I can run the docker commands without the option `network=host`. On the jenkins server, that hack is the only way I've found to make it work.

Reply:
OK, then let it as is